packages/server/postgres/migrations/1666387804873_userPatient0.ts

```diff
@@ -5,14 +5,18 @@ export async function up() {
   const client = new Client(getPgConfig())
   await client.connect()
   await client.query(`
-  ALTER TABLE "User"
-  ADD COLUMN IF NOT EXISTS "isPatient0" BOOLEAN DEFAULT FALSE;
-  `)
-  await client.query(`
-  UPDATE "User" u
-  SET "isPatient0" = COALESCE("createdAt" = (SELECT "createdAt" FROM "User" WHERE "domain" = u.domain ORDER BY "createdAt" LIMIT 1), false);
-  `)
-  await client.query(`
+    ALTER TABLE "User"
+    ADD COLUMN IF NOT EXISTS "isPatient0" BOOLEAN DEFAULT FALSE;
+
+    DROP TABLE IF EXISTS "tmp_patient0";
```

Comment:
+! out of curiosity, does it make sense to use `TEMPORARY` table here? 
https://www.postgresql.org/docs/current/sql-createtable.html 

they're session specific and were created for this exact use case

Reply:
Good suggestion! I will keep the `DROP` anyways, as the temp table isn't cleared up when running the query manually in pgadmin.